packages/client/src/client4.ts

```diff
@@ -3908,15 +3910,13 @@ export default class Client4 {
 
     trackEvent(category: string, event: string, props?: any) {
         if (this.telemetryHandler) {
-            const userRoles = this.userRoles && isSystemAdmin(this.userRoles) ? 'system_admin, system_user' : 'system_user';
-            this.telemetryHandler.trackEvent(this.userId, userRoles, category, event, props);
+            this.telemetryHandler.trackEvent(this.userId, this.userRoles, category, event, props);
```

Comment:
There's a slight change to the TelemetryHandler API here. We now pass it the raw user roles string (eg. `system_admin system_user`), and then we let the TelemetryHandler sort out what that means. Doing that let me get rid of the `isSystemAdmin` reference in here to avoid having to export any utility functions for the time being.

Ideally, this functionality wouldn't even be involved in the telemetry since it's sent through a separate client to a separate API. It doesn't actually make sense to have here.